### DISCUSSION TITLE
not receiving Hash code in OTP SMS

### DISCUSSION THREAD

**Discussion by gresi29** [firebase/firebase-android-sdk#2584] [Q&A]
I am working on an react native app that requires to verify users mobile number by sending an OTP. I am using Firebase phone authentication for this purpose. Everything working fine till I upload the app to play store. After uploading the app to the play store the SMS auto retrieve feature is no longer working.(the SMS doesn't contain the 11 digit hash key after uploading to play store).

below requirements are fulfilled

adding the sha-1 and sha-256 key from google play app-signing in play console
enable device check API in cloud console
enable mobile authentication in Firebase console

Any help is greatly appreciated as I am searching for this issue.

**Comment by emondd4** [comment]
did you find any solve?
